### ISSUE TITLE
Test with different settings on CI

### ISSUE THREAD

**Issue by mforets** [JuliaReach/LazySets.jl#1295]
(see [this comment](https://github.com/JuliaReach/LazySets.jl/pull/1289#discussion_r276522813))

The test suites are: `test_suite_basic`, `test_suite_doctests`, `test_suite_polyhedra`, `test_suite_plotting`.

**Comment by schillic** [comment]
Indeed, I noticed several problems already because we only test with all packages enabled.

Since the only relevant setting is `test_suite_polyhedra` (the others just skip subsets of the tests), it might actually be feasible to run the tests twice (because after the first run they have become compiled and are super fast).
